SimpsonsSwiftUIDemo/Classes/UI/Flows/Collection/CollectionView.swift

```diff
@@ -38,24 +38,24 @@ struct CollectionView: View {
         }
     }
 
-    private func collectionView(groups: [[SimpsonsEpisodeModel]]) -> some View {
+    private func collection_view(groups: [[SimpsonsEpisodeModel]]) -> some View {
         ScrollView {
             VStack {
                 ForEach(groups, id: \.self) { episodeGroup in
                     CollectionItemView(episodes: episodeGroup,
-                        episodeViewClicked: self.episodeViewClicked)
+                        episodeViewClicked: self.episode_view_tapped)
```

Comment:
But why its tapped? In our convention we decided to use generic "Clicked" naming.